docs/topics/azure-stack.md

```diff
@@ -0,0 +1,154 @@
+# AKS Engine on Azure Stack - Public Preview
+
+* [Introduction](#introduction)
+* [Service Principals and Identity Providers](#service-principals-and-identity-providers)
+* [CLI flags](#cli-flags)
+* [Cluster Definition (aka API Model)](#cluster-definition-aka-api-model)
+    * [location](#location)
+    * [kubernetesConfig](#kubernetesConfig)
+    * [customCloudProfile](#customCloudProfile)
+    * [masterProfile](#masterProfile)
+    * [agentPoolProfiles](#agentPoolProfiles)
+* [Azure Stack Instances Registered with Azure's China cloud](#azure-stack-instances-registered-with-azures-china-cloud)
+* [Disconnected Azure Stack Instances](#disconnected-azure-stack-instances)
+* [Unsupported Addons](#unsupported-addons)
+* [Known Issues and Limitations](#known-issues-and-limitations)
+* [Frequently Asked Questions](#frequently-asked-questions)
+
+## Introduction
+
+Starting from release v0.36.2, AKS Engine can be used to provision self-managed Kubernetes clusters on Azure Stack. It is now possible to execute AKS Engine's `generate`, `deploy`, `upgrade`, and `scale` commands as if you were targeting Azure's public cloud. You are only required to slightly update your cluster definition to provide some extra information about your Azure Stack instance.
```

Comment:
Maybe link "release v0.36.2" to https://github.com/Azure/aks-engine/releases/tag/v0.36.2 or https://github.com/Azure/aks-engine/releases/ ?